src/diffusers/loaders.py

```diff
@@ -135,6 +135,14 @@ def _fuse_lora(self, lora_scale=1.0):
             w_up = w_up * self.lora_linear_layer.network_alpha / self.lora_linear_layer.rank
 
         fused_weight = w_orig + (lora_scale * torch.bmm(w_up[None, :], w_down[None, :])[0])
```

Comment:
Question: do you know if usually the `nan` happens in
```python
w_orig + (lora_scale * torch.bmm(w_up[None, :], w_down[None, :])[0])
```

or in 
```python
(lora_scale * torch.bmm(w_up[None, :], w_down[None, :])[0])
```

cc @BenjaminBossan - if it happens in the second case we could remove the copy in the PEFT PR

Reply:
I don't know really. I have not tested it, but my intuition is that checking for NaN values can be quite expensive anyways when on GPU. So no matter what we have a time overhead and can't set `safe_fusing` as a default.

Reply:
OK sounds great!